tools/mob_suite/mob_recon.xml

```diff
@@ -12,78 +13,133 @@
     #set $named_input = re.sub(r'(\s|\(|\)|:|!)', '_', str($input.element_identifier)+'.fasta')
     ln -s '$input' '$named_input' &&
     
-    mob_recon  --num_threads \${GALAXY_SLOTS:-4}  --infile '${named_input}'
+
+    mob_recon  --num_threads \${GALAXY_SLOTS:-4}  --infile '${named_input}' --run_typer
+
     #if str($adv_param.unicycler_contigs) == "True":
       --unicycler_contigs 
     #end if   
-    #if str($adv_param.run_circlator) == "True":
-      --run_circlator 
-    #end if 
-    #if str($adv_param.min_length_condition.min_length_param) == "True":
-      --min_length '${adv_param.min_length_condition.min_length_value}'
+
+    #if str($adv_param.run_overhang) == "True":
+      --run_overhang
     #end if 
-    --run_typer --min_rep_evalue '${adv_param.min_rep_evalue}'
+    
+    #if $adv_param.debug:
+      --debug
+    #end if
+
+    #if $adv_param.plasmid_db
+   --plasmid_db $adv_param.plasmid_db
+   #end if
+
+   #if $adv_param.plasmid_mash_db
+   --plasmid_mash_db '$adv_param.plasmid_mash_db'
+   #end if
+
+   #if $adv_param.plasmid_meta
+   --plasmid_meta '$adv_param.plasmid_meta'
+   #end if
+
+   #if $adv_param.repetitive_mask
+   --repetitive_mask '$adv_param.repetitive_mask'
+   #end if
+
+   #if $adv_param.plasmid_mob
+   --plasmid_mob $adv_param.plasmid_mob
```

Comment:
Is `$adv_param.plasmid_mob` a free form field? If so, surround with `'`

Reply:
Done